.github/workflows/test_workflow_scripts/golang-docker.sh

```diff
@@ -80,10 +80,27 @@ for i in {1..2}; do
 
     echo "Recorded test case and mocks for iteration ${i}"
 done
+sleep 10
 
```

Comment:
this sleep isn't needed as send_request only exits when keploy record is stopped, so this would add unnecessary delay of 10 seconds